indico/modules/events/registration/blueprint.py

```diff
@@ -71,7 +72,12 @@
 
 _bp = IndicoBlueprint('event_registration', __name__, url_prefix='/event/<confId>', template_folder='templates',
                       virtual_template_folder='events/registration', event_feature='registration')
+_bp_participation = IndicoBlueprint('event_participation', __name__, url_prefix='/event/<confId>',
+                                    template_folder='templates', virtual_template_folder='events/registration')
 
+# Participants
+_bp_participation.add_url_rule('/manage/participants/',
+                               'manage_participants', RHManageParticipants, methods=('GET', 'POST'))
```

Comment:
I've been doing it like this for ages. If it doesn't fit in one line I put the name of the url_rule on a new line.